site/content/en/docs/tasks/troubleshooting/troubleshooting_jobs.md

```diff
@@ -60,6 +60,45 @@ Job is called `my-job` in the `my-namespace` namespace.
    job-my-job-19797   user-queue   cluster-queue   9m45s
    ```
 
+## Does my cluster-queue match the job resource requests?
+
+When you submit a job that has a resource request, for example:
+
+```bash
+$ kubectl get jobs job-0-9-size-6 -o json | jq -r .spec.template.spec.containers[0].resources
+```
+```console
+{
+  "limits": {
+    "cpu": "2"
+  },
+  "requests": {
+    "cpu": "2"
+  }
+}
+```
+
+If your cluster queue does not have a definition for the _requests_ the job cannot be admitted. For the job above you would want to ensure that you've defined "cpu" under `resourceGroups`, as shown below:
```

Comment:
```suggestion
If your ClusterQueue does not have a definition for the _requests_, Kueue cannot admit the job. For the job above, you would want to ensure that you've defined `cpu` under `resourceGroups`, as shown below:
```

Reply:
Although, I would try to avoid repeating too many things that are already described somewhere else. You can link to https://kueue.sigs.k8s.io/docs/concepts/cluster_queue/#resource-groups

The idea of this document is to give a decision tree for debugging, and link to the relevant documentation for the specifics.